Google/index.html

```diff
@@ -0,0 +1,157 @@
+<html>
+  <head>
+    <style>
+    header {height: 100%;}
+    body {
+        margin: 0;
+        padding: 0;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        }
+    * {font-family: sans-serif;}
+    nav {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding-left: 10px;
+        padding-right: 10px;
+        }
+    .nav-right {
+        display: flex;
+        align-items: center;
+        }
+    nav a, nav img {
+        margin-right: 15px;
+        }
+    a {
+        color: #333;
+        text-decoration: none;
+        }
+    a:hover {
+        text-decoration: underline;
+        }
+    main {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        }
+    .google-logo {width: 150px;}
+    .search-container {
+        display: flex;
+        align-items: center;
+        border: 1px solid #dfe1e5;
+        border-radius: 25px;
+        padding: 10px;
+        margin-top: 20px;
+        margin-bottom: 20px;
+        }
+    .search-container img {
+        width: 10px;
+         height: auto;
+        }
+    .search-container input {
+        margin-left: 10px;
+        margin-right: 10px;
+        width: 250px;
+        border: 0;
+        }
+    .button-container {
+        display: flex;
+        justify-content: center;
+        }
+    .button-container button {
+        margin-right: 5px;
+        background: #f2f2f2;
+        color: #333;
+        border: 0;
+        border-radius: 3px;
+        padding: 8px 16px;
+        }
+    .french {
+        text-align: center;
+        padding-top: 30px;         
+        }
+    footer  {
+        background: #f2f2f2;
+        border-top: solid 2px #e4e4e4;
+        bottom: 0;
+        padding-bottom: 0;
+        width: 100%;
+        }
+    footer ul li {
+        display: inline;
+        color: #666666;
+        font-size: 14px;
+        padding: 13px;
+        }
+    footer ul {
+        float: left;
+        padding: 1px;
+        }
+    footer ul a:hover {
+        text-decoration: underline;
+        }
+    .footer-right {
+        float: right;
+        }
+    .footer-ca {
+        margin-top: 13px;
+        margin-left: 13px;
+        margin-bottom: 13px;
+        }
+    </style>
+  </head>
+  <body>
+    <nav>
+      <div>
+        <a href="#">About</a>
+        <a href="#">Store</a>
+      </div>
+      <div class="nav-right">
+        <a href="#">Gmail</a>
+        <a href="#">Images</a>
+        <img src="menu.png" />
```

Comment:
This image should be little bit smaller to match other links size in same navigation.